server/src/test/java/com/ca/lsp/cobol/usecases/TestPerformWithoutEndNotCauseNPE.java

```diff
@@ -49,10 +49,10 @@ public void test() {
   @Override
   protected void assertDiagnostics(List<Diagnostic> diagnostics) {
     {
-      assertEquals(1, diagnostics.size(), "Number of diagnostics");
+      assertEquals(2, diagnostics.size(), "Number of diagnostics");
```

Comment:
Optional (all tests):  Hamcrest way - assertThat("Number of diagnostics", diagnostics, hasSize(2));

Reply:
I just found an issue with determining the error positions, so I'll fix it as a separate activity and rewrite the test